app/views/landing_page/blocks/_hero.html.erb

```diff
@@ -26,9 +30,10 @@
       <%= content_tag("div", class: grid_class) do %>
         <%= content_tag("div", class: hero_textbox_classes) do %>
           <% block.hero_content.each_with_index do |subblock, index| %>
-            <% options = {
-              position: block.data["position"],
-              margin_bottom: (0 if block.hero_content.length - 1 == index) } %>
+            <%
+              options = { margin_bottom: (0 if block.hero_content.length - 1 == index) }
```

Comment:
Can you clarify what this line is doing?

Reply:
It would be clearer if I did:

```ruby
is_final_subblock = block.hero_content.length - 1 == index
options = { margin_bottom: (0 if is_final_subblock) }
```

Reply:
Ah thanks I get it now. Maybe keep it as it is but add a comment? `# give last block zero margin bottom`?

Reply:
Done in b25bcdb

Reply:
Ah, sorry, I didn't see your comment until after I added that commit. I think it's better with the variable rather than a comment though - less likely to get left in place and become misleading if we change the code later.